ui/web/WEB-INF/templates/default.xhtml

```diff
@@ -32,171 +32,110 @@
                 <ui:include src="/WEB-INF/templates/fileupload.xhtml"/>
             </ui:insert>
             <h:form id="form1">
-                <div class="card">
```

Comment:
Please consider to check if sub parts of view can be extracted as composite components.